packages/mask/src/utils/theme/usePopupTheme.ts

```diff
@@ -1,14 +1,18 @@
 import { unstable_createMuiStrictModeTheme } from '@mui/material'
-import { MaskDarkTheme, MaskLightTheme } from '@masknet/theme'
+import { MaskDarkTheme, MaskLightTheme, useSystemPreferencePalette } from '@masknet/theme'
 import { languageSettings } from '@masknet/shared-base'
 import { useValueRef } from '@masknet/shared-base-ui'
 import { ThemeMode } from '@masknet/web3-shared-base'
-import { useThemeSettings } from '../../components/DataSource/useActivatedUI.js'
 import { useThemeLanguage } from './useThemeLanguage.js'
+import { useAppearance } from '../../../dashboard/pages/Personas/api.js'
```

Comment:
you should move this hook to somewhere else, for example, `packages/mask/shared-ui`. you should not import from dashboard directly.